evm/src/token-bridge-relayer/TokenBridgeRelayer.sol

```diff
@@ -19,26 +20,34 @@ import "./TokenBridgeRelayerMessages.sol";
 contract TokenBridgeRelayer is TokenBridgeRelayerGovernance, TokenBridgeRelayerMessages, ReentrancyGuard {
     using BytesLib for bytes;
 
+    // contract version
+    string public constant VERSION = "0.2.0";
```

Comment:
Will this be updated for major changes? Or for every deployment?

Reply:
We will update this for every deployment (at least the minor version number).